homeassistant/components/hive/light.py

```diff
@@ -40,73 +39,6 @@ async def async_setup_entry(
 class HiveDeviceLight(HiveEntity, LightEntity):
     """Hive Active Light Device."""
 
-    @property
-    def unique_id(self):
-        """Return unique ID of entity."""
-        return self._unique_id
-
-    @property
-    def device_info(self) -> DeviceInfo:
-        """Return device information."""
-        return DeviceInfo(
-            identifiers={(DOMAIN, self.device["device_id"])},
-            manufacturer=self.device["deviceData"]["manufacturer"],
-            model=self.device["deviceData"]["model"],
-            name=self.device["device_name"],
-            sw_version=self.device["deviceData"]["version"],
-            via_device=(DOMAIN, self.device["parentDevice"]),
-        )
-
-    @property
-    def name(self):
-        """Return the display name of this light."""
-        return self.device["haName"]
-
-    @property
-    def available(self):
-        """Return if the device is available."""
-        return self.device["deviceData"]["online"]
-
-    @property
-    def extra_state_attributes(self):
-        """Show Device Attributes."""
-        return {
-            ATTR_MODE: self.attributes.get(ATTR_MODE),
```

Comment:
Where are we setting extra state attributes now?

Reply:
I have removed it from the sensor list by mistake. We did put in the breaking change as part of this PR that it would be moved from a extra attribute to a sensor. Does that need to be in a separate PR or are we happy to add the sensor back in?

Reply:
We should not make any breaking changes or add new sensors here. Please put the `extra_state_attributes` back. We'll replace that in the next PR.

Reply:
Done